src/app/__test__/Rockets.test.js

```diff
@@ -0,0 +1,12 @@
+import renderer from 'react-test-renderer';
+import Rockets from '../__mocks__/Rockets';
+
+describe('Rockets', () => {
+  test('Rockets container items renders as expected', async () => {
+    const snapRockets = renderer.create(
+      <Rockets />,
+    )
+      .toJSON();
+    expect(snapRockets).toMatchSnapshot();
```

Comment:
line 10 expect(snapRockets).toMatchSnapshot(); 
this part of the code is not passing the test